### PR TITLE
Refactor and 1.3.2

### PR DESCRIPTION
**Pull request by the-best-elephant**
- Modify getOutputDir
- Ignore sphireTraining in protocol_cryolo_training

### CODE REVIEW

File: sphire/protocols/protocol_cryolo_picking.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def getInputModel(self):
         return os.path.abspath(m) if m else ''
 
     def getOutputDir(self):
-        return self._getTmpPath('outputEMAN')
+        return self._getExtraPath('Boxfiles')
```
Comment (delarosatrevin) [inline sphire/protocols/protocol_cryolo_picking.py]: Hi @the-best-elephant , why do you want to keep in 'extra' the temporary box files? They will be parsed and the SetOfCoordinates generated, so I think it is not needed.

Comment (pconesa) [reply sphire/protocols/protocol_cryolo_picking.py]: Hi @delarosatrevin , we were considering the box file not temporary, but the actual output of Cryolo (same as relion star files output, or xmipp). As such, we are doing the same, keeping the output of the program. Shouldn't we?

Comment (delarosatrevin) [reply sphire/protocols/protocol_cryolo_picking.py]: It is a temporary output that is parsed and no longer needed. I'm going the other way around in Relion and other plugins...removing files not needed.

**Comment by pconesa** [comment]
I always thought it was part of the scipion phylosophy. I liked the idea to
keep them for debug purposes. But I can see the burden of keeping all the
metadata files.

**Comment by delarosatrevin** [comment]
Yes, we have the philosophy of keeping original files...but in some case we generate more files than the job in the original package (e.g when using batches). So we need to be extra careful with additional  files and the garbage that is being accumulated. It is no good for parallel filesystems or for moving data around. 

For debugging purposes you can always activate the environment variable SCIPION_DEBUG_NOCLEAN=1 and the tmp folder will not be deleted. ;)

**Comment by delarosatrevin** [comment]
@pconesa , @the-best-elephant what is the current status of this PR? Do you still want to keep temporary files? Moreover, why do you commented out the training parameter in the training protocol?

**Comment by pconesa** [comment]
Forgot about this.....I still think is a good idea to keep metadata files but if reducing I/O become that important I'm ok to leave working dir on tmp.

On the training param...is there a point to have a pointer to another trainning in the trainning. We thought it was a legacy from a copy paste from the picking.

@the-best-elephant , why you've closed the PR?

**Comment by delarosatrevin** [comment]
@pconesa Even in training, it is possible to "fine tune" an existing model, that could be the general model or another one...so I think this option make sense. I'm not sure if it is properly implemented. 

BTW, I'm have tested (not this branch) with crYOLO 1.3.6 (lastest one at this moment) and seems to be working.

**Comment by pconesa** [comment]
Yep, we missunderstood it. We can leave it closed.

**Comment by the-best-elephant** [comment]
@delarosatrevin, @pconesa My bad. I will test the implementation.

**Pull request closed without merging**